doc/design.md

```diff
@@ -317,17 +452,22 @@ Database Schema
 
 Tables:
 
-    users (uuid, email, admin, date_created, date_modified), uuid primary key
+    users (uuid, email, name, group, admin, date_created, date_modified, activated), uuid primary key
 
     auth (uuid, hashed_password, user_id), uuid primary key
 
-    clients (uuid, first_name, last_name, dob, date_created, date_modified), uuid primary key
-
     result_codes (uuid, code) uuid primary key
 
     rules (uuid, text)
 
-    analyses (client_id, case_id, result_code, statute, date_eligible, rules[], date_created, date_modified, expunged, date_expunged)
+## Question:
+
+Do we store client information? I think we have at some point decided not to. 
```

Comment:
This is not really our decision to make. Of course it would make some things easier... But ultimately the client needs to sign off

Reply:
Ok that's what I had believed. Wanted to bring this up again before removing these tables from the design doc, but I think we can go ahead.